build/ci.mk

```diff
@@ -17,23 +18,23 @@ ci/debug-container: ci/deps
 			-e GPG_MAIL \
 			-e GPG_PASSPHRASE \
 			-e GPG_PRIVATE_KEY_BASE64 \
-			$(BUILDER_TAG) bash
+			$(BUILDER_IMAGE) bash
 
 .PHONY : ci/validate
 ci/validate: ci/deps
 	@docker run --rm -t \
 			--name "nri-$(INTEGRATION)-validate" \
 			-v $(CURDIR):/go/src/github.com/newrelic/nri-$(INTEGRATION) \
 			-w /go/src/github.com/newrelic/nri-$(INTEGRATION) \
-			$(BUILDER_TAG) make validate
+			$(BUILDER_IMAGE) make validate
```

Comment:
who is setting and then reading this environment variable? It comes from the makefile, and it is passed to a second makefile?

Reply:
ah ok I got it, it is replacing BUILDER_TAG